source/testing.html

```diff
@@ -0,0 +1,117 @@
+<!doctype html>
```

Comment:
Can we just remove this testing file all together now? I don't think you want anyone to accidentally stumble upon `http://poetspetparlor.com/testing.html` right?